docs/api-reference/layer-manager.md

```diff
@@ -45,14 +45,14 @@ Updates the current viewport.
 Updates parameters
 
 `layerManager.setParameters({
-	useDevicePixelRatio,
+	useDevicePixels,
     pickingRadius,
     onLayerClick,
     onLayerHover
 })
 `
 
-* `useDevicePixelRatio` (`Boolean`, optional) - Whether to use retina/HD display or not.
+* `useDevicePixels` (`Boolean`, optional) - Whether to use retina/HD display or not.
```

Comment:
For inter-framework consistency, the name of this parameter should presumably be changed in luma.gl `AnimationLoop` as well.

Reply:
Yep, mentioned that in description, will do that as a seperate PR.